doc/source/_toc.yml

```diff
@@ -63,25 +63,29 @@ parts:
             title: "Getting Started"
           - file: train/key-concepts
             title: "Key Concepts"
-          - file: train/user-guides
-            title: "User Guides"
+          - file: train/distributed-pytorch
+            sections:
+              - file: train/distributed-pytorch/quickstart
+              - file: train/distributed-pytorch/converting-existing-training-loop
+              - file: train/distributed-pytorch/configuration
+              - file: train/distributed-pytorch/data-loading-preprocessing
+              - file: train/distributed-pytorch/using-gpus
+              - file: train/distributed-pytorch/reporting-results
+              - file: train/distributed-pytorch/checkpoints
+              - file: train/distributed-pytorch/experiment-tracking-callbacks
+              - file: train/distributed-pytorch/fault-tolerance
+              - file: train/distributed-pytorch/hyperparameter-optimization
+                title: Hyperparameter optimization
+              - file: train/distributed-pytorch/advanced
+          - file: train/distributed-tensorflow-keras
+          - file: train/distributed-xgboost-lightgbm
+          - file: train/horovod
+          - file: train/internals/index
+            title: "Ray Train internals and advanced usage"
```

Comment:
This aggregation feels a bit unintuitive to me. Could we:

1. Make Architecture and Benchmarks top-level pages, and clean them up in the near future
2.  Remove the Predictors page (we are planning to de-emphasize the `Predictor` API in 2.7)

Reply:
I've removed the predictors page. I would still keep them in a subsection to not clutter the top level. I don't see why advanced internals like the architecture and benchmarks should be top-level when XGBoost/Tensorflow which are used in production are in a sublevel

Reply:
Sounds good. I'm not sure if benchmarks should fall under "Internals" but I'm also not strongly opposed to the current structure.